build.md

```diff
@@ -155,6 +155,40 @@ args:
   - GIT_COMMIT
 ```
 
+### ssh
+
+`ssh` defines SSH authentications that the image builder SHOULD use during image build (e.g., cloning private repository)
+
+`ssh` property syntax can be either:
+* `default` - let the builder connect to the ssh-agent.
+* `ID=path` - a key/value definition of an ID and the associated path. Can be either a [PEM](https://en.wikipedia.org/wiki/Privacy-Enhanced_Mail) file, or path to ssh-agent socket
+
+Simple`default` sample
+```yaml
+build:
+  context: .
+  ssh: 
+    - default   # mount the default ssh agent
+```
+or 
+```yaml
+build:
+  context: .
+  ssh: [default]   # mount the default ssh agent
```

Comment:
Should this be `["default"]`? I wouldn't expect that many readers would immediately get that this is an inline list with a single `default` string.